src/modules/previewpane/SvgPreviewHandler/Properties/Resource.resx

```diff
@@ -1,126 +1,126 @@
-﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
This shouldn't be here. Please revert it

Reply:
Which editor do you use to edit the file? It's probably autoformatting.
You can do something like;
```
git checkout origin/main <file>
```

and then commit the change

Reply:
Thanks worked